.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -673,3 +673,20 @@ cross_ResilientService:
       DescribeLoadBalancers, CreateTargetGroup, DescribeTargetGroups, DeleteTargetGroup,
       CreateLoadBalancer, CreateListener, DeleteLoadBalancer, DescribeTargetHealth}
     iam: {CreateInstanceProfile, DeleteInstanceProfile}
+cross_FMPlayground:
+  title: Create a Foundation Model Playground application that offers different playgrounds to interact with foundation models via &BR; API
+  title_abbrev: Create a playground application to invoke foundation models via &BR;
+  synopsis: create playgrounds to interact with foundation models via &BR; API through different modalities.
+  languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          block_content: cross_FMPlayground_NetV3_block.xml
+  service_main: bedrock
+  services:
+    bedrock:
+      {
+        ListFoundationModels,
+        InvokeModel,
+        InvokeModelWithResponseStream
+      }
```

Comment:
Add empty line (carriage-return from last line of code)
Required to parse correctly.

Reply:
I've applied the fix